index.js

```diff
@@ -3,7 +3,7 @@ var request = require('request');
 function buildJson(callback, label, currentJson, addressJson, currentIndex, finalIndex) {
 
   request({
-    url: "https://www.1secmail.com/api/v1/?action=readMessage&login="+label+"&domain=1secmail.com&id="+addressJson[currentIndex].id,
+    url: "https://www.1secmail.com/api/v1/?action=readMessage&login="+label+"&domain=wwjmp.com&id="+addressJson[currentIndex].id,
```

Comment:
I wouldn't want to hardcode a separate domain.

I'll add a commit that'll have a default list of domains and randomly select a domain from the list. It'll also be a parameter so you can modify the domains you want selected.